script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -571,109 +576,108 @@ I have no mental capacity left to allocate to any other eventualities.
 [sha:0df06b142239c91ae758079373e614b3eefa6017]{
 -- Page 693, Offset 10528.
 -- だからもう、降り始めた俺には<対|も><応|の>を考える機能はない。
-So now that I've started to descend,
- I don't have the ability to think about how to respond to him.
+Now that I've initiated the descent,
+ I've lost the ability to formulate thoughts about how to react to him.
```

Comment:
I no longer have the mental capacity to react to his actions.

Reply:
Duplicates the construction from the above line.

Reply:
Thinking about it, that's probably cue to change the above line, rather than this one. I like your suggestion.
JP
>それ以外の事象に振り分ける<意識|よぶん>はない。
PREV
>I have no mental capacity left to allocate to any other eventualities.
NEW
>There is nothing left in my consciousness to allocate to any other eventualities.